src/SoftDeleteable/Filter/SoftDeleteableFilter.php

```diff
@@ -71,7 +71,7 @@ public function addFilterConstraint(ClassMetadata $targetEntity, $targetTableAli
 
         $column = $quoteStrategy->getColumnName($config['fieldName'], $targetEntity, $platform);
 
-        $addCondSql = $platform->getIsNullExpression($targetTableAlias.'.'.$column);
+        $addCondSql = $targetTableAlias.'.'.$column . ' IS NULL';
```

Comment:
Can we confirm all the supported platforms allow this expression?

Reply:
From the link [above](https://github.com/doctrine/dbal/pull/4724) 
`They are not used by the DBAL itself. E.g. IS NULL is used as SQL everywhere instead of using getIsNullExpression().`

This is the deprecation message, and i followed the suggested replacement.
https://github.com/doctrine/dbal/pull/4724/files#diff-80f8e09bba3385ad6ffd160d565b52166f469a09b0c45b3cb0f93751490a3542R1068

Reply:
```suggestion
        $addCondSql = $targetTableAlias.'.'.$column.' IS NULL';
```